core/src/test/java/kafka/admin/ConfigCommandTest.java

```diff
@@ -16,10 +16,6 @@
  */
```

Comment:
It looks like there are a few unused methods in this file, for example `checkEntity()`, `checkInvalidArgs()`, `checkInvalidEntity()`